### PR TITLE
fix(rest): missing Group component in API documentation

### LINKED ISSUE #1900: Missing component Group in API documentation
### Description

While testing a new PR, I discovered following error in Swagger Editor.

image

Looks like the component information for `#/components/schemas/Group` was missed in #1872.

#### How to reproduce

Load the `src/www/ui/api/documentation/openapi.yaml` in Swagger editor.

### Versions

* Last commit id on master: 45e186a68fc3c3fbf42dad0a8e92dce3e39922f7

### PR DESCRIPTION
**Pull request by piotreke12**
Signed-off-by: Piotr Pszczola <piotr.pszczola@orange.com>

## Description

Fixes #1900 

### Changes

Added Group component to src/www/ui/api/documentation/openapi.yaml
Few naming fixes in src/www/ui/api/Models/Group.php

## How to test

Swagger integration verification should not show any errors

### CODE REVIEW

File: src/www/ui/api/documentation/openapi.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ openapi: 3.0.2
 info:
   title: FOSSology API
   description: Automate your fossology instance using REST API
-  version: 1.2.1
+  version: 1.2.2
```
Comment (GMishx) [inline src/www/ui/api/documentation/openapi.yaml:18]: I don't think it is necessary to update the API version. It is just a documentation patch. What do you think?

Comment (piotreke12) [reply src/www/ui/api/documentation/openapi.yaml:18]: This is a thing I also thought about. Finally I decided to change it as I don't know Your versioning policy very well. 
In fact, last part in semver policy says that last section in version number is for patch purposes, thats why I decided to change it.
All in all, I can revert this change if you wish so.

Comment (GMishx) [reply src/www/ui/api/documentation/openapi.yaml:18]: Yeah it is true that following semver policy, it should change to 1.2.2 but for the REST API, we tried to keep it simple.
1. If a change **does not** break backward compatibility (in terms of API call) but adds something new, we change the patch version (1.2.1 => 1.2.2).
2. If a change **does break** backward compatibility, we like to make it explicit by changing the minor version (1.2.1 => 1.3.0).
3. We have not yet faced or decided anything to change the major version of API :neutral_face:

Comment (GMishx) [reply src/www/ui/api/documentation/openapi.yaml:18]: So yeah, these changes does not do anything on the API so it should be okay to leave the version as is.

Comment (piotreke12) [reply src/www/ui/api/documentation/openapi.yaml:18]: Ok, Your explanation is clear.
Update reverting API version change is on it's way

**Review by GMishx — APPROVED** [review]
Changes looks good. Working fine with Swagger editor.

**Pull request merged**